services/ens/contracts.go

```diff
@@ -0,0 +1,63 @@
+package ens
+
+import (
+	"errors"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/rpc"
+	"github.com/status-im/status-go/services/ens/registrar"
+	"github.com/status-im/status-go/services/ens/resolver"
+)
+
+var resolversByChainID = map[uint64]common.Address{
+	1: common.HexToAddress("0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e"),
```

Comment:
Missing the chainId for Ropsten, Rinkeby and Goerli,  (the address is the same)

Reply:
Ah no wait. I'm confused. Here it says https://docs.ens.domains/ens-deployments that `0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e` is a registrar. Maybe it's a case of wrong variable name?